operations/enterprises.md

```diff
@@ -297,163 +300,284 @@ Returns all spaces of an enterprise associated with the connector integration.
+
+#### Resource data discriminator
+* `Space`
+
+#### Space resource data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `FloorNumber` | string | required | Number of the floor the space is on |
+
+## Update resources
+
+Updates resources.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/resources/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ResourceUpdates": [
+        {
+            "ResourceId": "5ee074b1-6c86-48e8-915f-c7aa4702086f",
+            "Name": {
+                "Value": "0101"
+            },
+            "ParentResourceId": null,
+            "Data": 
+            {
+                "Value": {
+                    "Discriminator": "Space",
+                    "Value": {
+                        "FloorNumber": "1"
+                    }
+                }
+            },
+            "State": {
+                "Value": "Clean"
+            },
+            "StateReason": {
+                "Value": "Sample reason"
+            }
+        }
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ResourceUpdates` | array of [Resource update](#resource-update) | required | Resource updates. |
+| `State` | string [Resource state](#resource-state) | required | New state of the resource \(`Dirty`, `Clean`, `Inspected` or `OutOfService`\). |
+
+#### Resource update
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ResourceId` | string | required | Unique identifier of the [Resource](#resource) which is updated. |
+| `Name` | [String update value](#string-update-value) | optional | New name of the resource \(e.g. room number\). |
+| `ParentResourceId` | [String update value](#string-update-value) | optional | Identifier of the new parent [Resource](#resource). |
+| `Data` | [Resource data update value](#resource-data-update-value) | optional | New additional data of the resource. |
+| `State` | [String update value](#string-update-value) | optional | New state of the resource \(`Dirty`, `Clean`, `Inspected` or `OutOfService`\). |
+| `StateReason` | [String update value](#string-update-value) | optional | New value of the rate on the interval. If not specified, removes all adjustments within the interval. |
+
+#### String update value
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Value` | string | optional | Value which is to be updated. |
+
+#### Resource data update value
```

Comment:
Why do we need update value for the data? Updating this should work differently - update values should be only on the properties inside the data, not on the whole object. This is not future proof and suffers from exactly same issues for which we introduced update values - if you introduce new property inside data, all clients updating floor number will suddenly start deleting the other property.